### PR TITLE
Update u8g2 doc (add extra online XBM format converter)

### PR DESCRIPTION
**Pull request by concefly**
add extra online XBM format converter

Fixes #\<GitHub-issue-number\>.

Make sure all boxes are checked (add x inside the brackets) when you submit your contribution, remove this sentence before doing so.

- [x] This PR is for the `dev` branch rather than for the `release` branch.
- [x] This PR is compliant with the [other contributing guidelines](https://github.com/nodemcu/nodemcu-firmware/blob/dev/CONTRIBUTING.md) as well (if not, please describe why).
- [x] I have thoroughly tested my contribution.
- [x] The code changes are reflected in the documentation at `docs/*`.

\<Description of and rationale behind this PR\>

### CODE REVIEW

**Comment by HHHartmann** [comment]
There is some kind of verification needed to upload an Image. My Chinese is not as good as that of Google but even they don't really have a clue. 
My feeling is that this service is free in terms of money only.

**Comment by concefly** [comment]
> There is some kind of verification needed to upload an Image. My Chinese is not as good as that of Google but even they don't really have a clue.
> My feeling is that this service is free in terms of money only.

Sorry I released this XBM online tool for Chinese developers mainly. It'is free. And I add some English note and release again.

Now, developers who use English can use it.

**Comment by HHHartmann** [comment]
Why Not give the Chinese community a hook here. It does seem to have problems with size changes though.

**Comment by concefly** [comment]
> It does seem to have problems with size changes though

OK I will update it continuously

File: docs/modules/u8g2.md [hunk 1]
```diff
@@ -155,7 +155,10 @@ Add the desired fonts to the font selection sub-entry via `make menuconfig`.
 ### Bitmaps
 XBM bitmaps are supplied as strings to `drawXBM()` in contrast to the source code based inclusion of XBMs in upstream u8g2 library. This off-loads all data handling from the u8g2 module to generic methods for binary files. See [graphics_test.lua](../../lua_examples/u8g2/graphics_test.lua).
 
-Conversion of XBM bitmaps can be performed online with [Online-Utility's Image Converter](http://www.online-utility.org/image_converter.jsp): Convert from XBM to MONO format and upload the binary result.
+Conversion of XBM bitmaps can be performed online with
+
+- [Online-Utility's Image Converter](http://www.online-utility.org/image_converter.jsp): Convert from XBM to MONO format and upload the binary result.
+- [XBM 格式生成器](https://tech.biko.pub/#/tool/xbm): Free online tool. Convert any image to XBM format (as lua module or C headers).
```
Comment (marcelstoer) [inline docs/modules/u8g2.md:161]:
```suggestion
- [XBM 格式生成器](https://tech.biko.pub/#/tool/xbm): Free online tool. Convert any image to XBM format (as Lua module or C headers).
```

**Pull request merged**